example/otlp/README.md

```diff
@@ -0,0 +1,80 @@
+# OTLP Example
+This example demonstrates how to export trace and metric data from an
+application using OpenTelemetry's own wire protocol
+[OTLP](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/protocol/README.md).
+We will also walk you through configuring a collector to accept OTLP exports.
+
+### How to run?
+
+#### Prequisites
+- go >=1.14 installed
```

Comment:
This repo should remain compatible with the current and prior versions of Go, so 1.13 and 1.14 at this time.  Is it possible to eliminate any dependency on 1.14?

Reply:
Yup! Fixed